src/docs/01-get-started/01-server-installation.md

```diff
@@ -6,70 +6,33 @@ permalink: /docs/get-started/installation
-                                 Dload  Upload   Total   Spent    Left  Speed
-100  1264  100  1264    0     0   4461      0 --:--:-- --:--:-- --:--:--  4698
-  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
-                                 Dload  Upload   Total   Spent    Left  Speed
-100    14  100    14    0     0     71      0 --:--:-- --:--:-- --:--:--    76
-> ls
-docker-compose.yml      prometheus
+curl -O https://raw.githubusercontent.com/uber/cadence/master/docker/docker-compose.yml && curl -O https://raw.githubusercontent.com/uber/cadence/master/docker/prometheus/prometheus.yml
 ```
-Start Cadence Service:
+Then start Cadence Service by running:
 ```bash
-> docker-compose up
-Creating network "quick_start_default" with the default driver
-Pulling cadence (ubercadence/server:0.5.8)...
-0.5.8: Pulling from ubercadence/server
-db0035920883: Pull complete
-...
-...
-27ec3755f89c: Pull complete
-0a5d2a29a5e5: Pull complete
-Creating quick_start_statsd_1    ... done
-Creating quick_start_cassandra_1 ... done
-Creating quick_start_cadence_1   ... done
-Creating quick_start_cadence-web_1 ... done
-Attaching to quick_start_cassandra_1, quick_start_statsd_1, quick_start_cadence_1, quick_start_cadence-web_1
-statsd_1       | *** Running /etc/my_init.d/00_regen_ssh_host_keys.sh...
-statsd_1       | *** Running /etc/my_init.d/01_conf_init.sh...
-cadence_1      | + CADENCE_HOME=/cadence
-cadence_1      | + DB=cassandra
-...
-...
-...
-cadence_1      | {"level":"info","ts":"2019-06-06T15:26:38.199Z","msg":"Get dynamic config","name":"matching.longPollExpirationInterval","value":"1m0s","default-value":"1m0s","logging-call-at":"config.go:57"}
-cadence_1      | {"level":"info","ts":"2019-06-06T15:26:38.199Z","msg":"Get dynamic config","name":"matching.updateAckInterval","value":"1m0s","default-value":"1m0s","logging-call-at":"config.go:57"}
-...
-...
-cadence_1      | {"level":"info","ts":"2019-06-06T15:27:24.905Z","msg":"Get dynamic config","name":"history.timerProcessorCompleteTimerFailureRetryCount","value":"10","default-value":"10","logging-call-at":"config.go:57"}
+docker-compose up
 ```
-## Register a Domain Using the CLI
-From a different console window:
+Please keep this process running at background.
+
+## 2. Register a Domain Using the CLI
+In a new terminal, create a new domain called `test-domain` (or choose whatever name you like) by running:
 ```bash
-> docker run --network=host --rm ubercadence/cli:master --do test-domain domain register -rd 1
-Unable to find image 'ubercadence/cli:master' locally
-master: Pulling from ubercadence/cli
-22dc81ace0ea: Pull complete
-...
-...
-da2cfe74be81: Pull complete
-5320bde81c0c: Pull complete
-Digest: sha256:f5e5e708347909c8d3f74c47878b201d91606994394e94eaede9a80e3b9f077b
-Status: Downloaded newer image for ubercadence/cli:master
-Domain test-domain successfully registered.
->
+docker run --network=host --rm ubercadence/cli:master --do test-domain domain register -rd 1
```

Comment:
Just wanted to make sure, by having the docker cmds in the front, we don't need to start a Cadence server right? (Like: `./cadence-server --zone elasticsearch start`)